HISTORY.rst

```diff
@@ -2,6 +2,27 @@
 History
 =======
 
+3.66.0 (2022-09-29)
+-------------------
+
+Changes 🎉
+~~~~~~~~~~
+* New Metadata ETL task ``set_object_settings`` to enable and disable settings on standard and custom objects (#3364)
+* For robot tests on the firefox profile now accept all cookies to prevent lightning pages from failing to render due to cross-domain cookies. (#3366)
```

Comment:
```suggestion
* For Robot tests on the Firefox profile, we now accept all cookies to prevent lightning pages from failing to render due to cross-domain cookies. (#3366)
```